eventuate-core/src/main/scala/com/rbmhtechnology/eventuate/ReplicationEndpoint.scala

```diff
@@ -592,6 +593,8 @@ private class FailureDetector(sourceEndpointId: String, logName: String, failure
     case FailureDetected(cause) =>
       causes = causes :+ cause
     case FailureDetectionLimitReached(scheduledCount) if scheduledCount == counter =>
+      log.error(causes.lastOption.getOrElse(Logging.Error.NoCause), "replication failure detection limit reached ({})," +
+        " publishing Unavailable for {}/{} (last exception being reported)", failureDetectionLimit, sourceEndpointId, logName)
```

Comment:
another nitpick: avoid splitting the string with `+` and start the new line before `"replication...`